### PR TITLE
fix: Includes 90° x-axis label rotation

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Includes 90° x-axis label rotation by default, as opposed to requiring users to input the value, as it's frequently used for charts with many data points.

Check https://github.com/apache/superset/issues/26159 for more context.

### TESTING INSTRUCTIONS
CI should be sufficient.

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by sfirke — APPROVED** [review]
LGTM.  Does this cover bar charts?  I didn't see "bar" on the page but maybe it's called something else.

**Comment by michael-s-molina** [comment]
> LGTM. Does this cover bar charts? I didn't see "bar" on the page but maybe it's called something else.

Now it does 😆

File: superset-frontend/plugins/plugin-chart-echarts/src/Bubble/controlPanel.tsx [hunk 1]
```diff
@@ -138,6 +138,7 @@ const config: ControlPanelConfig = {
               choices: [
                 [0, '0°'],
                 [45, '45°'],
+                [90, '90°'],
```
Comment (john-bodley) [inline superset-frontend/plugins/plugin-chart-echarts/src/Bubble/controlPanel.tsx:141]: @michael-s-molina is the last choice assigned the default?

Comment (john-bodley) [reply superset-frontend/plugins/plugin-chart-echarts/src/Bubble/controlPanel.tsx:141]: Sorry I misread your PR description. I initially read it as you were setting `90°` as the default as opposed to adding it as one of the default options.

File: superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/Step/controlPanel.tsx [hunk 2]
```diff
@@ -184,6 +184,7 @@ const config: ControlPanelConfig = {
               choices: [
                 [0, '0°'],
                 [45, '45°'],
+                [90, '90°'],
```
[the remark this answers is not in the document]
Comment (michael-s-molina) [reply superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/Step/controlPanel.tsx:187]: This is just one example of the many code duplication in the ECharts plugins. I'm planning to improve all this once the migrations are complete.

[the remark this answers is not in the document]
Comment (villebro) [reply superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/Step/controlPanel.tsx:187]: If centralizing this now is easy (which I assume it should be), I'd probably just do this right away to get it out of the way. But if you prefer to do it later that's also ok.

**Review by villebro — COMMENTED** [review]
Approving to unblock, but I'd still recommend centralizing the control to leave the codebase slightly DRYer

edit: oops, I didn't notice this was already merged 😆

**Pull request merged**